app/server/models/transfer_account.py

```diff
@@ -190,8 +190,10 @@ def get_approval(self, receiving_address):
 
     def approve_and_disburse(self, initial_disbursement=None):
         from server.utils.access_control import AccessControl
-
-        active_org = getattr(g, 'active_organisation', self.primary_user.default_organisation)
+        primary_user_default_org = getattr(self.primary_user, 'default_organisation')
+        active_org = getattr(g, 'active_organisation', primary_user_default_org)
```

Comment:
If there was no `g` object (like in the context of tests), and the transfer account has no user, this gave a key error.